docs/developers/tutorials/staking-contract.md

```diff
@@ -353,35 +334,18 @@ This is NOT an error. This simply means you provided way more gas than needed, s
 
 ## The first stake
 
-Let's add a snippet for the staking function:
+Let's call the stake function:
 
 ```bash
-USER_PEM="~/Downloads/tutorialKey.pem"
-PROXY="https://devnet-gateway.multiversx.com"
-CHAIN_ID="D"
-
-SC_ADDRESS=erd1qqqqqqqqqqqqq...
-STAKE_AMOUNT=1
-
-deploy() {
-    mxpy --verbose contract deploy --project=${PROJECT} \
-    --recall-nonce --pem=${USER_PEM} \
+mxpy --verbose contract call ${SC_ADDRESS}... \
```

Comment:
Here, I think we can use a hardcoded, well-defined SC address (already deployed on devnet), then instruct the user to replace it with their own contract address, as resulted upon running the deploy command.